src/rpc/handlers/LedgerEntry.h

```diff
@@ -100,13 +100,17 @@ class LedgerEntryHandler
         static auto const rpcSpec = RpcSpec{
             {JS(binary), validation::Type<bool>{}},
             {JS(ledger_hash), validation::Uint256HexStringValidator},
-            {JS(ledger_index), validation::LedgerIndexValidator},
+            {JS(ledger_index),
```

Comment:
For "ledger_index", rippled returns "invalidParams". no need to change this one

Reply:
How did it make into the discrepancy list then 😆